mteb/tasks/Classification/multilingual/ScalaClassification.py

```diff
@@ -1,167 +1,56 @@
-        reference="https://aclanthology.org/2023.nodalida-1.20/",
-        dataset={
-            "path": "mteb/scala_nn_classification",
-            "revision": "d81637ad324afb995ae395a87055380e8118a9c0",
-        },
-        type="Classification",
-        category="s2s",
-        eval_splits=["test"],
-        eval_langs=["nno-Latn"],
-        main_score="accuracy",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
-        n_samples={"test": 1024},
-        avg_character_length={"test": 104.8},
-    )
-
-    @property
-    def metadata_dict(self) -> dict[str, str]:
-        metadata_dict = super().metadata_dict
-        metadata_dict["n_experiments"] = 10
-        metadata_dict["samples_per_label"] = 32
-        return metadata_dict
-
-    def dataset_transform(self):
-        # convert label to a 0/1 label
-        labels = self.dataset["train"]["label"]  # type: ignore
-        lab2idx = {lab: idx for idx, lab in enumerate(set(labels))}
-        self.dataset = self.dataset.map(
-            lambda x: {"label": lab2idx[x["label"]]}, remove_columns=["label"]
-        )
+_LANGS = {
+    "Danish": ["dan-Arab"],
+    "Norwegian_b": ["nob-Hans"],
+    "Norwegian_n": ["nno-Latn"],
+    "Swedish": ["swe-Latn"],
+}
 
 
-class ScalaSvClassification(AbsTaskClassification):
+class MultiScalaClassification(AbsTaskClassification, MultilingualTask):
     metadata = TaskMetadata(
-        name="ScalaSvClassification",
-        description="A Swedish dataset for linguistic acceptability classification",
+        name="MultiScalaClassification",
+        description="""ScaLa - multilingual version of nordic languages dataset for linguistic acceptability classification.
+        Published as part of 'ScandEval: A Benchmark for Scandinavian Natural Language Processing'""",
         reference="https://aclanthology.org/2023.nodalida-1.20/",
         dataset={
-            "path": "mteb/scala_sv_classification",
-            "revision": "aded78beae37445bf3917102d0b332049cfc8c99",
+            "path": "mteb/multilingual-scala-classification",
+            "revision": "ec85bb6c69679ed15ac66c0bf6e180bf563eb137",
         },
         type="Classification",
         category="s2s",
         eval_splits=["test"],
-        eval_langs=["swe-Latn"],
+        eval_langs=_LANGS,
         main_score="accuracy",
         date=None,
-        form=None,
+        form=["written"],
         domains=None,
-        task_subtypes=None,
+        task_subtypes=["Linguistic acceptability"],
         license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
         dialect=None,
```

Comment:
```suggestion
        dialect=[],
```
None as far as I am aware